vector/src/main/java/im/vector/riotx/core/ui/views/NotificationAreaView.kt

```diff
@@ -77,34 +63,39 @@ class NotificationAreaView @JvmOverloads constructor(
         when (newState) {
             is State.Default                    -> renderDefault()
             is State.Hidden                     -> renderHidden()
+            is State.NoPermissionToPost         -> renderNoPermissionToPost()
             is State.Tombstone                  -> renderTombstone(newState)
-            is State.ResourceLimitExceededError -> renderResourceLimitExceededError(newState)
```

Comment:
`ResourceLimitExceededError` case will have to be restored one day, which is not the case for the other ones (so thanks for the cleanup). I would have kept ResourceLimitExceededError then.